SPECS/vim/vim.spec

```diff
@@ -184,6 +189,8 @@ fi
 %{_bindir}/vimdiff
 
 %changelog
+*   Thu Oct 15 2020 Emre Girgin <mrgirgin@microsoft.com> 8.1.0388-7
+-   Fix CVE-2019-12735, CVE-2019-20079, CVE-2019-20807.
```

Comment:
Does it fix all these CVEs? If so, don't wee need nopatches for the others?

Reply:
Good catch, that was changelog was incorrect.